### DISCUSSION TITLE
DOM for the e-commerce

### DISCUSSION THREAD

**Discussion by BacLuc** [ase-23-the-bald-owls/documentation#7] [Ideas]
Here i found some UML diagrams for the DOM of an e-commerce site:

https://itsourcecode.com/uml/e-commerce-website-uml-diagrams-complete/
https://www.uml-diagrams.org/examples/online-shopping-domain-uml-diagram-example.html
https://www.gyaanibuddy.com/assignments/assignment-detail/uml-diagrams-for-ecommerce/

![image](https://user-images.githubusercontent.com/1506818/223121915-86665681-0355-44aa-9b8b-1e7179ee5005.png)
[edit link](http://www.plantuml.com/plantuml/uml/TLFRQW8n47tFLmpRDwc2hoA2zAM5bWhI3yWcgHlCZIHRaT9_xoHcirvzC9dBENDc9iiK3G4stlAUHVWbWByfsf89SEc_UBRHIxC3G_MLH_AAoHpRI6E2ZaQ9Vr2tF66UJySAQCHBsqSHIeuq4KAqR0zFraLf3LKvCflDgE6WhHVaZn2MHNzwBdoLdArJSLkl0_WH-YqANHbISTFhWQ-4poYhUPVMMCppSjT5iDyZ_M9nXLr3wDl0lCmDWXAK1uWMTA-YTAhK0ec008TEiWuODT0A29rqJd08mb6FLEf8XE6fBXcwUkbnVFfiB6LBhLGAppNdNeJmUAORUjJ2f8TGNL-Xq5KoeQbKvmw7UGUK_Kf3b_tlCiJbi6Ll5Iwuycs5KhS2RnyMwjtV4qfEUs_c3IodqnKqasQSDXdNcX9o7C_oDWVeP81YQ9WO6LeVYUU5cEGHNFWhgdHpeQPKMwIlu4ToRFuc2o-9bFm3)

**Comment by Off3line** [comment]
Looking good for a first draft version!
